### PR TITLE
Proofread. Update with minor edits 2016-10-10

### PR DESCRIPTION
**Pull request by FedericoEsparza**
Great reflection on this post. 

I have some additional question comments about a few lines.

### CODE REVIEW

File: source/blog_articles/2016-10-10-multidimensional-learning-economics.html.markdown [hunk 1]
```diff
@@ -11,23 +11,23 @@ So anyway, I usually have a rule of not doing any work on the weekends, but I ha
...
+I want to get learners comfortable about asking in public so that others can benefit.  If they ask in group chat on AgileVentures it increases the activity in the community, and the community value keeps people around, and the rising tide lifts all the boats.  I could devote all my time to answering 121 questions, and I might get a few thanks from people, and I might learn a few things, but it's not enough to feed my kids.  It makes me think that the new AgileVentures premium tier should be called "121", cost about £25 a month, and get you an assigned mentor who can answer your questions on 121 chat.  And maybe a £50 a month version called Premium on Air that gets you an hour of hangout time.  We'll have to work out the details.  We need to avoid changing and tweaking too much or we confuse people, muddy the water.  It really is a complex "multidimensional economics of learning" problem ...
```
Comment (FedericoEsparza) [inline source/blog_articles/2016-10-10-multidimensional-learning-economics.html.markdown]: Do the quotes in the last line maintain your intent for the line, or is there another way to rework it, so it's more clear?

File: source/blog_articles/2016-10-10-multidimensional-learning-economics.html.markdown [hunk 2]
```diff
@@ -11,23 +11,23 @@ So anyway, I usually have a rule of not doing any work on the weekends, but I ha
...
 
-It was holding my feet to the learning fire that got me my AstroPhysics degree.  It was so tempting to keep reading, and re-summarizing my notes.  It wasn't until I forced myself to repeatedly answer old exam questions that I got my exam technique down and got the first that got me on to higher education.  Not that I was ever really satisified with myself during my first masters and PhD.  It still felt trivial and isolated from the rest of the world.  Part of the problem was that I was studying in the dark ages before search engines like Google connected learners with the rest of the world.
+It was holding my feet to the learning fire that got me my AstroPhysics degree.  It was so tempting to keep reading and re-summarizing my notes.  It wasn't until I forced myself to repeatedly answer old exam questions that I got my exam technique down and got the first that got me on to higher education.  Not that I was ever really satisified with myself during my first masters and PhD.  It still felt trivial and isolated from the rest of the world.  Part of the problem was that I was studying in the dark ages before search engines like Google connected learners with the rest of the world.
```
Comment (FedericoEsparza) [inline source/blog_articles/2016-10-10-multidimensional-learning-economics.html.markdown]: I think you might have omitted a word between "got the first" and "that got".

File: source/blog_articles/2016-10-10-multidimensional-learning-economics.html.markdown [hunk 3]
```diff
@@ -11,23 +11,23 @@ So anyway, I usually have a rule of not doing any work on the weekends, but I ha
...
+There seems to be a prevaling view that taking courses equals education.  I guess I'm drinking my own kool-aid here, but I feel like learning comes from doing.  More and more courses are hands on these days, but I'm skeptical of the learning benefits of watching lots of lectures, or even toy "exercuses".  I guess it's a relatively low impact way of floating your brain in some of the domain concepts.  Not everyone wants to become an expert in a particular field.  However if you're in a rush to skill up and you can take the emotional pain of making lots of mistakes (really valuable for learning), then I think diving in to a real project is the way to go.
```
Comment (FedericoEsparza) [inline source/blog_articles/2016-10-10-multidimensional-learning-economics.html.markdown]: Was "exercuses" intentional or a typo?

File: source/blog_articles/2016-10-10-multidimensional-learning-economics.html.markdown [hunk 4]
```diff
@@ -11,23 +11,23 @@ So anyway, I usually have a rule of not doing any work on the weekends, but I ha
...
 
-Now AgileVentures doesn't provide as much of a pathway as FreeCodeCamp or CodeCademy or even CodeWars, and all of these still have some beautifully unstructured portions.  We're still working on getting the balance right at AgileVentures, and also the economic model to make is sustainable.  It seems like both CodeWars and CodeCademy are leaning towards being solutions to help employers find job candidates, and FreeCodeCamp is operating as a free for all non-profits and developers, while charging companies a hiring fee when FCC developers are placed (like many bootcamps).
+Now AgileVentures doesn't provide as much of a pathway as FreeCodeCamp or CodeCademy or even CodeWars, and all of these still have some beautifully unstructured portions.  We're still working on getting the balance right at AgileVentures, and also the economic model to make it sustainable.  It seems like both CodeWars and CodeCademy are leaning towards being solutions to help employers find job candidates, and FreeCodeCamp is operating as a free choice for all non-profits and developers, while charging companies a hiring fee when FCC developers are placed (like many bootcamps).
```
Comment (FedericoEsparza) [inline source/blog_articles/2016-10-10-multidimensional-learning-economics.html.markdown]: I'm not sure if inserting "choice" fits the intent of the line. Verify that that's the case.

**Comment by tansaku** [comment]
thanks @FedericoEsparza - fixed as per your tips

**Pull request merged**